packages/amplify-appsync-simulator/package.json

```diff
@@ -49,24 +49,17 @@
     "moment": "^2.24.0",
     "moment-jdateformatparser": "^1.2.1",
     "moment-timezone": "0.5.34",
-    "mqtt-connection": "4.0.0",
```

Comment:
In terms of removing MQTT support, does this mean we are no longer supporting local testing for IoT?  Do customers use this behavior? If so, what is our guidance for these customers with this change?

Reply:
I don't think the usage of MQTT here is to support IoT use-cases. From the data that I could find, the older implementations in Amplify Client Libraries allowed MQTT based AppSync subscriptions before shifting to use real-time websockets. Refer to [this comment](https://github.com/aws-amplify/amplify-cli/issues/3655#issuecomment-599662423) and [this one](https://github.com/aws-amplify/amplify-cli/issues/3008#issuecomment-599169057). I've tried to keep both of these approaches, but ran into issues because both processes need to use the same port. I believe that is the reason @yuth removed it too in his [draft PR](https://github.com/aws-amplify/amplify-cli/pull/9709).

Reply:
Yep. The first iteration of AppSync was MQTT based, fully migrated in January, we're deprecating the MQTT AppSync provider with the next major version bump. MQTT is also offered as a PubSub provider for non-amazon MQTT servers and then is the protocol for IoT integrations on the PubSub category.